### PR TITLE
Fix incorrect section reference as pointed out in LWG 2181 which beca…

### PR DESCRIPTION
**Pull request by Dani-Hub**
…me not accepted during Kona meeting because of this being considered an editorial issue

### CODE REVIEW

File: source/numerics.tex [hunk 1]
```diff
@@ -3187,7 +3187,7 @@
 
 \pnum
 Each type instantiated
-from a class template specified in this section~\ref{rand.eng}
```
Comment (tkoeppe) [inline source/numerics.tex]: Why do we need a reference here at all?

**Comment by Dani-Hub** [comment]
The style to specify the concrete section makes the context clearer and we use this wording form in several places of the library.

**Comment by tkoeppe** [comment]
Makes sense, thanks.

**Comment by tkoeppe** [comment]
Could you please rename the commit message to include the section reference?

**Comment by jwakely** [comment]
Fixed at 45201e86de1f68c2ddb84b1dc90d961e97a9195f with a better commit message :)

**Pull request closed without merging**